source/content/guides/local-development/04-ddev.md

```diff
@@ -38,29 +33,75 @@ Be sure that you have:
 
 1. Follow the system prompts to install DDEV.
 
+1. Choose and install a [supported Docker provider](https://ddev.readthedocs.io/en/latest/users/install/docker-installation/) for use with DDEV, then open the program on your machine before you proceed. 
+
 ## Use DDEV with Pantheon
 
 1. Navigate to your Pantheon Dashboard and [generate a machine token](/machine-tokens/) for use with your DDEV site.
 
-1. Open your global DDEV configuration file at `~/.ddev/global_config.yaml` and add the API token to the `web_environment` section:
+1. Open your global DDEV configuration file at `~/.ddev/global_config.yaml` and add your Pantheon machine token to the `web_environment` section, replace `insertyourtoken` with your token value:
 
-    ```bash{promptUser: user}
+    ```yaml
     web_environment:
-    - TERMINUS_MACHINE_TOKEN=insertyourtoken
+        - TERMINUS_MACHINE_TOKEN=insertyourtoken
     ```
 
-1. Copy the `example.pantheon.yaml` provider file to the `pantheon.yaml`.
+1. [Clone the site's codebase](/guides/git/git-config#clone-your-site-codebase), and from the site's root directory run: 
+
+  ```bash{promptUser: user}
+  ddev config
+  ```
+
+1. Start your DDEV project locally by running the following command. 
+
+  ```bash{promptUser: user}
+  ddev start
+  ```
+
+  In addition to starting local Docker containers for the site, this command will also install DDEV provider integration recipes to your site's codebase at `.ddev/providers`, which we will use in the next step.
+
+1. Copy your site's `.ddev/providers/example.pantheon.yaml` provider file to `.ddev/providers/pantheon.yaml`.
+
+  <Alert title="Note" type="info" >
+
+  Do this in your site's `.ddev` directory, not the global `.ddev` directory. The `.ddev/providers/pantheon.yaml` file is a DDEV recipe for integration with Pantheon. For more information on DDEV provider integrations, see [DDEV docs](https://ddev.readthedocs.io/en/latest/users/providers/).
 
-1. Update the project name and environment variable. In the example below, the project name is `de8` and the targeted environment is `live`. This example updates the local DDEV environment with database and content file backups from the Pantheon Live environment.
+  </Alert>
 
-    ```bash{promptUser: user}
+1. Update the project name and environment variable in your new `.ddev/providers/pantheon.yaml` file. In the example below, the project name is `de8` and the targeted environment is `dev`.
```

Comment:
Might want to be more specific here. It's not obvious to me whether `d8.dev` is a `<site>.<env>` Pantheon environment or a ddev-specific local domain (e.g. `https://d8.dev`) (something that's slightly more obvious when the example was `d8.live`).

```suggestion
1. Update the project name and environment variable in your new `.ddev/providers/pantheon.yaml` file. In the example below, the Pantheon project name is `de8` and the targeted Pantheon environment is `dev`.
```